package.json

```diff
@@ -4,23 +4,24 @@
   "main": "index.js",
   "author": "@geoctrl",
   "license": "MIT",
+  "repository": "git@github.com:CanopyTax/kremling-loader.git",
```

Comment:
This will add a link to https://www.npmjs.com/package/kremling-loader for people to find the github repo